File: src/main/java/gg/mineral/practice/commands/settings/NightCommand.java
package gg.mineral.practice.commands.settings;

import org.bukkit.entity.Player;

import gg.mineral.practice.commands.PlayerCommand;
import gg.mineral.practice.managers.ProfileManager;
import gg.mineral.practice.util.messages.impl.ChatMessages;

public class NightCommand extends PlayerCommand {

    public NightCommand() {
        super("night");
    }

    @Override
    public void execute(Player player, String[] args) {
        ChatMessages.TIME_SET_NIGHT.send(player);
        player.setPlayerTime(14000L, false);
        ProfileManager.getOrCreateProfile(player).setNightMode(true);
    }
}

File: src/main/kotlin/gg/mineral/practice/commands/settings/NightCommand.kt
package gg.mineral.practice.commands.settings

import dev.rollczi.litecommands.annotations.command.Command
import dev.rollczi.litecommands.annotations.context.Context
import dev.rollczi.litecommands.annotations.execute.Execute
import gg.mineral.practice.entity.appender.CommandSenderAppender
import gg.mineral.practice.util.messages.impl.ChatMessages
import org.bukkit.entity.Player

@Command(name = "night")
class NightCommand : CommandSenderAppender {
    @Execute
    fun execute(@Context player: Player) {
        player.send(ChatMessages.TIME_SET_NIGHT)
        player.setPlayerTime(14000L, false)
    }
}